pcap/pcap.go

```diff
@@ -362,6 +362,11 @@ func (p *Handle) ZeroCopyReadPacketData() (data []byte, ci gopacket.CaptureInfo,
 		slice.Cap = ci.CaptureLength
 	}
 	p.mu.Unlock()
+	if slice.Len != len(slice.Data) {
```

Comment:
I'm guessing this doesn't compile?  The 'slice' variable shouldn't be referencable here.  Also, note that slice.Data is NOT a slice.  It's a uintptr, so len(slice.Data) makes no sense.